process-engine/documentation
# Remove deprecated documentation components

## What did you change?

Remove the documents which were marked as `deprecated`.

## How can others test the changes?

Best view in gitbook and see that the deprecated components are no longer accessible.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
